wire-ios-data-model/Source/MLS/MessageProtocol.swift

```diff
@@ -16,8 +16,6 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import Foundation
```

Comment:
As far as I know `Foundation` contains all the base layer things like `String`. Should this import be removed?
For Example when you move things to a Swift Package, it is required to have some `import` statement, like `import Swift` for a minimum.